tests/Resources/fs.addontest.js

```diff
@@ -0,0 +1,1493 @@
+			fs.access(thisFilePath, fs.constants.X_OK, err => {
+				try {
+					should(err).be.ok; // aka, there is an error
+					// TODO Verify the error.code value is EACCESS!
+					finished();
+				} catch (e) {
+					finished(e);
+				}
+			});
+		});
+	});
+
+	describe('#accessSync()', () => {
+		it('is a function', () => {
+			should(fs.accessSync).be.a.Function;
+		});
+
+		it('checks that this file exists properly', () => {
+			fs.accessSync(thisFilePath, fs.constants.F_OK);
+		});
+
+		it('throws when trying to access file that doesn\'t exist', () => {
+			should.throws(() => {
+				fs.accessSync('/madeup');
+			}, Error);
+			// TODO: Verify format of error!
+		});
+
+		it('uses mode F_OK by default', () => {
+			fs.accessSync(thisFilePath);
+		});
+
+		it('checks that this file is readable properly', () => {
+			fs.accessSync(thisFilePath, fs.constants.R_OK);
+		});
+
+		it.allBroken('checks that this file is NOT writable properly', () => {
+			// FIXME: This isn't throwing an error like we expect!
+			// How can we test this? resourcesDirectory is not supposed to be writable on device, but can be on simulator.
+			// (and this may be platform-specific behavior)
+			should.throws(() => {
+				fs.accessSync(thisFilePath, fs.constants.W_OK);
+			}, Error);
+			// TODO Verify the error.code value is EACCESS!
+		});
+
+		it('checks that this file is NOT executable properly', () => {
+			should.throws(() => {
+				fs.accessSync(thisFilePath, fs.constants.X_OK);
+			}, Error);
+			// TODO Verify the error.code value is EACCESS!
+		});
+	});
+
+	// TODO: #appendFile()
+
+	describe('#appendFileSync()', () => {
+		it('is a function', () => {
+			should(fs.appendFileSync).be.a.Function;
+		});
+	});
+
+	// TODO: #close()
+
+	describe('#closeSync()', () => {
+		it('is a function', () => {
+			should(fs.closeSync).be.a.Function;
+		});
+
+		it('returns undefined', () => {
+			const fd = fs.openSync(thisFilePath);
+			const result = fs.closeSync(fd);
+			should(result).be.undefined();
+		});
+	});
+
+	describe('#copyFile()', () => {
+		it('is a function', () => {
+			should(fs.copyFile).be.a.Function;
+		});
+
+		it('copies file asynchronously to destination', function (finished) {
+			this.slow(2000);
+			this.timeout(5000);
+
+			const dest = path.join(Ti.Filesystem.tempDirectory, 'fs.addontest.js');
+			// ensure file doesn't already exist
+			const destFile = Ti.Filesystem.getFile(dest);
+			if (destFile.exists()) {
+				should(destFile.deleteFile()).eql(true);
+			}
+			should(destFile.exists()).eql(false);
+
+			fs.copyFile(thisFilePath, dest, err => {
+				try {
+					should(err).not.be.ok;
+					fs.existsSync(dest).should.eql(true);
+					// TODO: Read in the file and compare contents? Check filesize matches?
+					finished();
+				} catch (e) {
+					finished(e);
+				}
+			});
+		});
+	});
+
+	describe('#copyFileSync()', () => {
+		it('is a function', () => {
+			should(fs.copyFileSync).be.a.Function;
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/f...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/fs.addontest.js;
  Line: 206;
-->

- :warning: tests/Resources/fs.addontest.js line 206 – The 'fs.copyFileSync' is not supported until Node.js 8.5.0. The configured version range is '>=8'. (node/no-unsupported-features/node-builtins)